tests/integration/ovs_test.py

```diff
@@ -103,3 +117,44 @@ def test_create_and_remove_ovs_bridge_with_internal_port_and_static_ip():
 
     assertlib.assert_absent(BRIDGE1)
     assertlib.assert_absent(PORT1)
+
+
+@pytest.mark.xfail(raises=NmstateLibnmError, strict=True)
+def test_nm_ovs_plugin_missing():
+    with disable_nm_ovs_plugin():
+        libnmstate.apply(
+            {
+                Interface.KEY: [
+                    {
+                        Interface.NAME: BRIDGE1,
+                        Interface.TYPE: InterfaceType.OVS_BRIDGE,
+                        Interface.STATE: InterfaceState.UP,
+                    }
+                ]
+            }
+        )
+
+
+@contextmanager
+def disable_nm_ovs_plugin():
+    libcmd.exec_cmd(DNF_REMOVE_NM_OVS_CMD)
+    libcmd.exec_cmd(SYSTEMCTL_RESTART_NM_CMD)
+    libnmstate.nm.checkpoint._nmdbus_manager = None
+    ret = [1, 0, 0]
+    timeout = 0
+    while ret[0] or timeout > 20:
```

Comment:
The condition for the `timeout` variable should be inversed and the name is not correct (the timeout is IMHO the 20 not the elapsed time). So better use something like: `while ret[0] or elapsed_time < 20`

Reply:
Yes, you are right. I used `while ret[0] != 0 and elapsed_time < 20`. Thanks!